ddl/concurrentddltest/switch_test.go

```diff
@@ -70,8 +70,9 @@ func TestConcurrentDDLSwitch(t *testing.T) {
 		}
 	}
 
-	c := atomic.NewInt32(0)
-	ch := make(chan struct{})
+	ddlDoneCnt := atomic.NewInt32(0)
```

Comment:
this test would be removed soon.

Reply:
OK, so no issues with updating it for now then :)

Reply:
FYI: https://github.com/pingcap/tidb/pull/39684, it's in merging.

Reply:
Thanks, merging those changes here now.

Reply:
Just merged, thank you for the heads up!